### ISSUE TITLE
Missing form labels

### ISSUE THREAD

**Issue by torgeirl** [devilry/trix2#134]
The form control for search field in the header on course admin pages does not have a corresponding label. This violates success criteria 3.3.2 in WCAG 2.1.

**Comment by Levijatan** [comment]
Fixed in 0b9bfc0, also fixed an empty <th> assignment multiselect view
